fe/fe-core/src/main/java/com/starrocks/connector/unified/UnifiedMetadata.java

```diff
@@ -50,15 +50,21 @@ public class UnifiedMetadata implements ConnectorMetadata {
     public static final String ICEBERG_TABLE_TYPE_VALUE = "iceberg";
     public static final String SPARK_TABLE_PROVIDER_KEY = "spark.sql.sources.provider";
     public static final String DELTA_LAKE_PROVIDER = "delta";
+    public static final String PAIMON_STORAGE_HANDLER_KEY = "storage_handler";
+    public static final String PAIMON_STORAGE_HANDLER_VALUE = "org.apache.paimon.hive.PaimonStorageHandler";
 
-    static boolean isIcebergTable(Map<String, String> properties) {
+    public static boolean isIcebergTable(Map<String, String> properties) {
         return ICEBERG_TABLE_TYPE_VALUE.equalsIgnoreCase(properties.get(ICEBERG_TABLE_TYPE_NAME));
     }
 
-    static boolean isDeltaLakeTable(Map<String, String> properties) {
+    public static boolean isDeltaLakeTable(Map<String, String> properties) {
         return DELTA_LAKE_PROVIDER.equalsIgnoreCase(properties.get(SPARK_TABLE_PROVIDER_KEY));
     }
 
+    public static boolean isPaimonTable(Map<String, String> properties) {
+        return PAIMON_STORAGE_HANDLER_VALUE.equalsIgnoreCase(properties.get(PAIMON_STORAGE_HANDLER_KEY));
+    }
+
```

Comment:
what about paimon? just defined and no usage. Left LGTM

Reply:
reserved for later usage